lms/static/js/learner_dashboard/models/certificate_model.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Model for Course Programs.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ],
+        function (Backbone) {
+            return Backbone.Model.extend({
+                initialize: function(data) {
+                    if (data){
+                        this.set({
+                            display_name: data.display_name,
+                            url:data.credential_url
+                        });
```

Comment:
There should not be a space after the word function, it should be ```function(data) {```. There should be for the other 2 though.